src/components/pages/AddFarmer/AddFarmer.jsx

```diff
@@ -0,0 +1,160 @@
+/** @format */
+
+import React, {useState} from 'react'
+import Input from '../../common/Input/Input'
+import {personalInfo, familyInfo, guarantor, farmInfo} from '../../common/Input/addFarmerData'
+import Button from '../../common/Button/StyledButton'
+import styled from 'styled-components'
+import axios from 'axios'
+
+const AddFarmer = () => {
+  const [state, setState] = useState({
+    personalInfo: personalInfo,
+    familyInfo: familyInfo,
+    guarantor: guarantor,
+    farmInfo: farmInfo,
+  })
+  const [stateToggle, setStateToggle] = useState({
+    personalInfoToggle: false,
+    familyInfoToggle: true,
+    guarantorToggle: true,
+    farmInfoToggle: true,
+  })
+
+  const onChangeHandler = (e, data) => {
+    const {name, value, type} = e.target
+    const newData = {...state[data]}
+    const newEntry = {...newData[name]}
+    if (type === 'checkbox') {
+      if (newEntry.selected.indexOf(value) > -1) {
+        newEntry.selected = newEntry.selected.filter(s => s !== value)
+      } else {
+        newEntry.selected = [...newEntry.selected, value]
+      }
+    } else {
+      newEntry.value = value
+    }
+    newData[name] = newEntry
+    setState({...state, [data]: newData})
+  }
```

Comment:
Formatting can use a bit of work. Could you add an empty line after each handler to make for more logical separation when reading the code?